src/plugins/kbn_vislib_vis_types/index.js

```diff
@@ -2,10 +2,8 @@ module.exports = function (kibana) {
 
   return new kibana.Plugin({
 
-    uiExports: {
-      visTypes: [
-        'plugins/kbn_vislib_vis_types/kbn_vislib_vis_types'
-      ]
+    ui: {
+      visTypes: './kbn_vislib_vis_types'
```

Comment:
Yeah, I wouldn't block this PR on it or anything.

Personally, I vote for always leaving off the .js extension.  If we leave off extension, then we are free to evolve the module from a single file into a directory with an `index.js` without changing all calls to it.  That sort of change is kind of irrelevant to the code that's invoking the module, and the module loader handles it transparently.